src/go/k8s/tests/e2e-v2/resource-redpanda-crud/01-update-chart-version.yaml

```diff
@@ -5,4 +5,7 @@ metadata:
   name: redpanda
 spec:
   chartRef:
-    chartVersion: "4.0.13"
\ No newline at end of file
+    chartVersion: "4.0.13"
+  clusterSpec:
+    statefulset:
+      replicas: 1
```

Comment:
Why this needs to be added?

Reply:
Shouldn't we test 3 node Redpanda cluster?

Reply:
Yeah we can, i do not see the reason to, but ill revert.